src/Microsoft.DotNet.Interactive.Jupyter/KernelExtensions.cs

```diff
@@ -22,10 +22,14 @@ namespace Microsoft.DotNet.Interactive.Jupyter
 {
     public static class KernelExtensions
     {
-        public static Task UseJupyterHelpersAsync<TKernel>(this TKernel kernel) where TKernel : DotNetKernel
+        public static void UseJupyterHelpers<TKernel>(this TKernel kernel) where TKernel : DotNetKernel
         {
             var interactiveHost = new JupyterInteractiveHost();
-            return kernel.SetVariableAsync("InteractiveHost", interactiveHost, typeof(IInteractiveHost));
+            var command = new AnonymousKernelCommand((_command, context) =>
+            {
+                return kernel.SetVariableAsync("InteractiveHost", interactiveHost, typeof(IInteractiveHost));
```

Comment:
does this address the completion too? we can do the using statements too here if we defer those commands

Reply:
Instead of an anonymous command, you can now do `KernelInvocationContext.ScheduleAsync`.